cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -112,9 +117,9 @@ ExternalProject_Add(
     "@CMAKE_COMMAND@" -P
     "@HUNTER_GLOBAL_SCRIPT_DIR@/append-boost-config-macros.cmake"
     "@HUNTER_Boost_CMAKE_ARGS@"
+    PATCH_COMMAND
```

Comment:
Logically it belongs to `CONFIGURE_COMMAND`. Also note that order of `UPDATE_COMMAND` and `PATCH_COMMAND` is not specified:
* https://gitlab.kitware.com/cmake/cmake/merge_requests/1178#note_308268

Probably it doesn't make any difference here but I think it's better not to mix them.